lib/functions.sh

```diff
@@ -2164,9 +2164,11 @@ make_clean() {
         CLEAN_SOURCE=
         return
     fi
+
+    eval set -- $MAKE_CLEAN_ARGS_WS
     logmsg "--- make (dist)clean"
     (
-        $MAKE distclean || $MAKE clean
+        $MAKE $MAKE_CLEAN_ARGS distclean || $MAKE $MAKE_CLEAN_ARGS clean
```

Comment:
You set the args list from the whitespace variant (`MAKE_CLEAN_ARGS_WS`) but then don't use it. Add `$@` like this to use the result of that:

```suggestion
        $MAKE $MAKE_CLEAN_ARGS "$@" distclean \
            || $MAKE $MAKE_CLEAN_ARGS "$@" clean
```

Can you also add these variables to `lib/config.sh`, where the other MAKE_ARGS ones are?